lib/vanagon/platform/dsl.rb

```diff
@@ -43,6 +44,8 @@ def platform(name, &block)
                       Vanagon::Platform::Solaris10.new(@name)
                     when /^solaris-11/
                       Vanagon::Platform::Solaris11.new(@name)
+                    when /^win-/
```

Comment:
We're using the short name in the platform definitions: `win-2012-x64.rb`. Although we could change this.